### PR TITLE
Replace non-general codes (Yinst -> Install)

### PR DESCRIPTION
**Pull request by quiye**
Hi,
I replaced non-general codes using `Yinst` with generic `Install`, by referring to #2896 .

### CODE REVIEW

**Comment by bratseth** [comment]
@freva can you take a look at the CoreCollector and Maintainer changes?

The rest are 👍   Thanks @quiye!

**Review by freva — APPROVED** [review]
The `Maintainer`/`CoreCollector` changes are fine, but they need to be coordinated with some changes in internal repos and this requires at least 1 release. I'll merge this once its ready.

**Comment by quiye** [comment]
@bratseth @freva Thank you for reviewing!

**Pull request merged**
